addon/components/o-s-s/input-container.hbs

```diff
@@ -29,6 +29,18 @@
       <div class="suffix">{{yield to="suffix"}}</div>
     {{/if}}
   </div>
+  {{#if @successMessage}}
+    <span class="text-color-success margin-top-px-6">
+      <OSS::Icon @icon="fa-check-circle" />
+      <span>{{@successMessage}}</span>
+    </span>
+  {{/if}}
+  {{#if @alertMessage}}
+    <span class="text-color-alert margin-top-px-6">
```

Comment:
font-color-warning-500 instead (the text-color-X classes are legacy)

Reply:
i guess you based it off the text-color-error which is old. just don't introduce new ones ;) 
(also don't change the error one because we most likely use it to target error messages in tests)